### ISSUE TITLE
Successfully implemented PySwarms in my regression problem!

### ISSUE THREAD

**Issue by JaveJao** [ljvmiranda921/pyswarms#46]
Good day, sir! I just want to thank you for helping me in my assignment. I really appreciate you and most especially, PySwarms <3 I really love it! I am happy that I was able to implement PySwarms in a regression problem using MSE as the loss function. I will not be able to accomplish it if it weren't for your kindness. Once again, thank you very much and may you have a bright future 👍

**Comment by ljvmiranda921** [comment]
Hi @JaveJao, I'm happy that you were able to implement a neural network using MSE on your own. Congratulations to you as well! 🥇 Hope it piqued your interest more in machine learning and optimization. Glad that I became of help. Goodluck to your studies and I hope that you achieve your dreams. 👍
